tooling/src/main/java/org/opencds/cqf/tooling/operation/ig/CqlRefresh.java

```diff
@@ -0,0 +1,98 @@
+package org.opencds.cqf.tooling.operation.ig;
+
+import org.hl7.fhir.instance.model.api.IBaseResource;
+import org.opencds.cqf.tooling.parameter.RefreshIGParameters;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class CqlRefresh extends Refresh {
+
+    private static final Logger logger = LoggerFactory.getLogger(CqlRefresh.class);
+    private final Pattern VERSION_PATTERN = Pattern.compile("^(library\\s+(\\S+)\\s+version\\s+)'[0-9]+\\.[0-9]+\\.[0-9]+'");
```

Comment:
Not sure whether this would be easier, but it seems like it would definitely be more robust to use the Antlr visitor framework for this?